sources/whippet/src/networkmanager.rs

```diff
@@ -0,0 +1,144 @@
+use serde::Deserialize;
+use zbus::{dbus_proxy, Result};
+
+#[dbus_proxy(
+    default_service = "org.freedesktop.network1",
+    interface = "org.freedesktop.network1.Manager",
+    default_path = "/org/freedesktop/network1",
+    assume_defaults = false
+)]
+trait NetworkManager {
```

Comment:
I'm on the fence about including all of these.  I do appreciate the effort it took to gather this information and the fact we _could_ use them, but I think we should probably whittle this down to what we _do_ use.  I would like to save them for future reference and use, though...  Perhaps we comment them out with a message "for potential future use" or something?

Reply:
I agree we should trim this down. These are autogenerated from `zbus_xmlgen` so we can always go back and get them again. I think I can trim these down as long as I document where we got them and how one would efficiently add things back in.

Reply:
I pulled out many of the things we are unlikely to use (the write stuff) but left some of the other read functions since its still early and I realized we might have nicer ways to do this. For example, we might shift to `get_link_by_name` vs `list_links`. I also anticipate us finding edge cases where the `*_state` calls might be needed. We can trim as we discover they are not needed as we test this code more.